src/components/Header.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+
+export function Header(): JSX.Element {
+  return (
+    <header className="p-3 mb-3 align-right" style={{ background: '#009cab' }}>
+      <div className="container">
+        <HeaderLink to="/">
+          <HeaderLogo />
+        </HeaderLink>
+
+        <nav className="float-right">
+          <HeaderLink to="/">Tracks</HeaderLink>
```

Comment:
For semantics, consider adding a list, as you're showing a list of links:

```
<nav>
  <ul>
    <li><link /></li>
```

You can use `list-unstyled`

Reply:
See https://github.com/exercism/v3-dashboard/issues/93